web/src/components/base/ToggleButton.tsx

```diff
@@ -1,53 +1,46 @@
 import styled from '@emotion/styled'
-import { SvgIconProps } from '@mui/material'
+import { SvgIconProps, ToggleButton as MuiToggleButton } from '@mui/material'
 import React, { ElementType, ReactElement } from 'react'
 
 import StyledSmallViewTip from '../StyledSmallViewTip'
-import Button from './Button'
 import Icon from './Icon'
 
 export const toggleButtonWidth = 100
 
-const StyledIcon = styled(Icon)`
-  width: 40px;
-  height: 40px;
-  color: ${props =>
-    props.theme.isContrastTheme ? props.theme.colors.backgroundColor : props.theme.colors.textSecondaryColor};
+const StyledIcon = styled(Icon)<{ iconSize: 'small' | 'medium' }>`
```

Comment:
❓ @hauf-toni would it make sense to unify the icon size here? Currently it looks like this:
![image](https://github.com/user-attachments/assets/81400fc2-9924-4a35-9632-01057f1a2f58)
With same-size icons compared to the poi filters, it would like this:
![image](https://github.com/user-attachments/assets/df107364-5bdf-49bd-a102-08d160972499)
For comparison, the poi filters look like this:
![image](https://github.com/user-attachments/assets/9f3b2a17-da9f-40b2-88fb-53f58e9d3e75)